app/services/plan_service/data_types.rb

```diff
@@ -18,6 +20,8 @@ module PlanService::DataTypes
     [:marketplace_plan_id, :fixnum, :mandatory],
     [:marketplace_id, :fixnum, :mandatory],
     [:plan_level, :fixnum, :mandatory],
+    [:features, :hash, :mandatory],
+    [:member_limit, :fixnum, :optional],
```

Comment:
This `ExternalPlan` entity is admittedly badly named. It is used to pass initial trial data to Charm in the login link JSON Web Token payload. `features` and `member_limit` (and actually also `plan_level`) are unnecessary information and could (should?) be removed from the payload.